apps/omg_watcher/test/db/transaction_test.exs

```diff
@@ -24,6 +24,22 @@ defmodule OMG.Watcher.DB.TransactionTest do
 
   require Utxo
 
+  describe "get_by_filters/3" do
```

Comment:
Please revert, already tested [here](https://github.com/omisego/elixir-omg/blob/master/apps/omg_watcher/test/omg_watcher/web/controllers/transaction_test.exs)

Reply:
But those are unit tests for the `Transaction` schema 🤔Controller tests are not enough to ensure that each function at the schema level works as expected and is thoroughly tested. It also makes sense to build a pyramid, a lot of small unit tests testing a lot of edge cases at the bottom (which I didn't really do because I'm quite unfamiliar with the fixture approach you're using and didn't want to mess up everything), and a few high level integration/controller tests testing the happy path.

Reply:
yeah, we relied on controller tests, since the whole thing was super-simplistic, now as it evolves we can reconsider, in case the controller tests get too unwieldy.